arc-script/src/symbols.rs

```diff
@@ -0,0 +1,148 @@
+use crate::ast::*;
+use crate::error::CompilerError;
+use DeclKind::*;
+
+/// The symbol-table is used to store symbolic identifiers for functions, let-expressions, 
```

Comment:
Almost all non-code lines in this comment adds trailing whitespace.

Reply:
Oops, fixed!